.env.example

```diff
@@ -74,17 +74,19 @@ MAX_FILE_SIZE_MB=80
 OPA_API_CLIENT_MP_REST_URL=http://localhost:8181
 # Open Notifications API secret key
 OPEN_NOTIFICATIONS_API_SECRET_KEY=openNotificatiesApiSecretKey
+# Enables or disables the support for creating a documenta via SmartDocuments.
+SMARTDOCUMENTS_ENABLED=false
 # SmartDocuments API authentication token. Please contact SmartDocuments to request an API token.
-SD_AUTHENTICATION=XXX
+SMARTDOCUMENTS_AUTHENTICATION=XXX
```

Comment:
Maybe too late now but I would not change existing env vars in this already large PR. I would have just add the new `SMARTDOCUMENTS_ENABLED` env var and leave all the others to `SD_xxx` for now and separately discuss how we want to go about renaming those in a separate change.

Reply:
For now it's probably ok but for future I think we need to take a lot care in breaking changes in the Helm Charts. It requires a separate process.